sdk/digitaltwins/Azure.DigitalTwins.Core/tests/prerequisites/setup.ps1

```diff
@@ -76,121 +68,61 @@ $appId = az ad app list --show-mine --query "[?displayName=='$AppRegistrationNam
 if (-not $appId)
 {
     Write-Host "`nCreating App Registration $AppRegistrationName`n"
-    $appId = az ad app create --display-name $AppRegistrationName --native-app --required-resource-accesses $pathToManifest --query 'appId' --output tsv
+    $appId = az ad app create --display-name $AppRegistrationName --native-app --query 'appId' --output tsv
 }
 
-$spExists = az ad sp list --show-mine --query "[?appId=='$appId'].appId" --output tsv
-if (-not $spExists)
+$sp = az ad sp list --show-mine --query "[?appId=='$appId'].appId" --output tsv
+if (-not $sp)
 {
     Write-Host "`nCreating service principal for app $appId`n"
     az ad sp create --id $appId --output none
 }
 
+# Get test application OID from the service principal
+$applicationOId = az ad sp show --id $sp --query "objectId" --output tsv
+
 $rgExists = az group exists --name $ResourceGroup
 if ($rgExists -eq "False")
 {
     Write-Host "`nCreating Resource Group $ResourceGroup in $Region`n"
     az group create --name $ResourceGroup --location $Region --output none
 }
 
-$dtExists = az dt list -g $ResourceGroup --query "[?name=='$DigitalTwinName']" --output tsv --only-show-errors
-if (-not $dtExists)
-{
-    Write-Host "`nCreating Digital Twin $DigitalTwinName`n"
-    az dt create -n $DigitalTwinName -g $ResourceGroup --location $Region --output none --only-show-errors
-}
-$dtHostName = az dt show -n $DigitalTwinName -g $ResourceGroup --query 'hostName' --output tsv --only-show-errors
-
-# The Service Principal takes a while to get propogated and if a different endpoint is hit before that, trying to grant a permission will fail.
-# Adding retries so that we can grant the permissions successfully without re-running the script.
-Write-Host "Assigning owner role to $appId for $DigitalTwinName`n"
-$tries = 0;
-while (++$tries -le 10)
-{
-    try
-    {
-        az dt role-assignment create --assignee $appId --role owner -n $DigitalTwinName --output none --only-show-errors
-
-        if ($LastExitCode -eq 0)
-        {
-            Write-Host "`tSucceeded"
-            break
-        }
-    }
-    catch { }
-
-    if ($tries -ge 10)
-    {
-        Write-Error "Max retries reached for granting service principal permissions."
-        throw
-    }
-
-    Write-Host "`tGranting service principal permission failed. Waiting 5 seconds before retry..."
-    Start-Sleep -s 5;
-}
+Write-Host "`nDeploying resources to $ResourceGroup in $Region`n"
 
-Write-Host("Set a new client secret for $appId`n")
-$appSecret = az ad app credential reset --id $appId --years 2 --query 'password' --output tsv
+$armTemplateFile = "../../../test-resources.json";
```

Comment:
If someone runs the script, but from a different directory, this will fail. You need to Join-Path with $PSScriptRoot.